.github/workflows/stale_repos.yml

```
@@ -24,5 +24,5 @@ jobs:
       with:
         title: Stale repository report
         content-filepath: ./STALE_REPOS.md
-        assignees: @jeffpaul
+        assignees: "jeffpaul"
```

Looking at the example workflow they provide, it seems it should be the username without the `@` symbol and looks like without quotes either: https://github.com/peter-evans/create-issue-from-file/blob/main/.github/workflows/ciff-example-command.yml#L36
```suggestion
        assignees: jeffpaul
```